### DISCUSSION TITLE
Waiting for response when sending from PC to mobile

### DISCUSSION THREAD

**Discussion by SPLESHER1** [localsend/localsend#2062] [Discussion (English, International)]
Hello,
I've recently encountered an issue with transferring files from my PC to my mobile device. Essentially, it freezes on the screen that indicates 'waiting for response.' I've attempted restarting the server, router, rebooting both the phone and PC, and reinstalling the applications. A few days ago, everything was functioning perfectly fine and sending from mobile to PC works flawlessly.
![2024-11-19 17_18_35-LocalSend](https://github.com/user-attachments/assets/1f734b01-ff8a-4dad-99bb-c99d6d8c84cb)

**Comment by NolanNolan** [comment]
I have exactly the same problem. Uninstalled. So annoying that its almoset standard that apps dont work as supposed. A shame, the idea is good.

**Comment by NolanNolan** [comment]
OK seems setting timeout to 1000 solved it

Thanks

Nolan

**Reply by SPLESHER1** [reply]
Sadly that didn't work for me, I had to change port to different one on both PC and mobile, but whenever I get new device with localsend i will need to change port everytime

**Reply by NolanNolan** [reply]
I also got the error again . So still no solution to this. 
I hope it will be fixed asap, its otherwise a great app

**Reply by SPLESHER1** [reply]
> I also got the error again . So still no solution to this. I hope it will be fixed asap, its otherwise a great app

You can just change port to different one, I set it on 52319 until it's fixed, but you need to change on both devices as I said earlier

**Comment by NolanNolan** [comment]
i tried to set a new port in all apps ie on windows, and 2 android devices. Still dont work

this needs a fast fix, as it is now its unfortunately useless

thanks
